enterprise/api/v1/views.py

```diff
@@ -1483,3 +1483,42 @@ def update(self, request, *args, **kwargs):
     )
     def destroy(self, request, *args, **kwargs):
         return super().destroy(request, *args, **kwargs)
+
+class EnterpriseUserLinkView(EnterpriseReadWriteModelViewSet):
+    """
+    View for
+    /enterprise/api/enterprise_user_link/{enterprise_customer_key}
+
+    Given a enterprise_customer_key, link user to the appropriate enterprise.
+
+    If the key is not found, returns 400
+    If the ket is not valid, return 422
+    """
+    permission_classes = (permissions.IsAuthenticated,)
+
+    @action(methods=['post'], detail=False)
+    def link_user(self, request, enterprise_customer_key):
+        """
+        Post
+        Links user using enterprise_customer_key
+        """
+        try:
+            enterprise_customer_key_match = models.EnterpriseCustomerInviteKey.objects.get(pk=enterprise_customer_key)
```

Comment:
is `enterprise_customer_key` a uuid? even if the EnterpriseCustomerInviteKey's primary key is a uuid, i think it would be clearer to use `uuid=enterprise_customer_key` instead of `pk=enterprise_customer_key`